lib/param_parsing/versiontf.go

```diff
@@ -0,0 +1,26 @@
+package param_parsing
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"log"
+)
+
+func GetVersionFromVersionsTF(params Params) Params {
+	filePath := params.ChDirPath + "/version.tf"
+	if lib.CheckFileExist(filePath) {
+		fmt.Printf("Reading version from %s\n", filePath)
+		module, err := tfconfig.LoadModule(params.ChDirPath)
+		if err != nil {
+			log.Fatal("Could not load terraform module")
+		}
+		tfconstraint := module.RequiredCore[0]
+		version, err2 := lib.GetSemver(tfconstraint, params.MirrorURL)
```

Comment:
Would it make sense to re-use `err` instead of defining a new `err2`?

Reply:
err is another type than err2. I tried that 😄

Reply:
Odd Go 🤣